src/patch/parse.js

```diff
@@ -7,41 +7,38 @@ export function parsePatch(uniDiff, options = {}) {
     let index = {};
     list.push(index);
 
-    // Ignore any leading junk
-    while (i < diffstr.length) {
-      if (/^(Index:|diff -r|@@)/.test(diffstr[i])) {
+    // Parse diff metadata
+    for (; i < diffstr.length; i++) {
```

Comment:
This is not purely stylistic, this is to mimic the same style done on [line 78](https://github.com/kpdecker/jsdiff/pull/88/files#diff-a74b5a56ea1528866a8b97a54a3dfd6bR78) on this same file to have an uniform style on both places.